### PR TITLE
Add /tools/dart-devtools

### PR DESCRIPTION
**Pull request by kwalrath**
Also removed the placeholder for an overview page that doesn't seem necessary.

### CODE REVIEW

File: src/tools/dart-devtools.md [hunk 1]
```diff
@@ -0,0 +1,16 @@
+---
+title: Dart DevTools
+description: A suite of performance tools for Dart and Flutter.
+toc: false
+---
+
+Dart DevTools is a suite of performance tools for Dart and Flutter.
+DevTools supports source-level debugging of Dart command-line apps,
```
Comment (mit-mit) [inline src/tools/dart-devtools.md]: There is nothing command-line specific about DevTools. You can use it for Flutter apps, command-line apps, server apps, and web apps. I suggest we just say 'Dart apps'.

File: src/tools/dart-devtools.md [hunk 2]
```diff
@@ -0,0 +1,16 @@
...
+as well as viewing general log and diagnostics information about a running app.
+
+We expect developers to use DevTools as part of their existing
```
Comment (mit-mit) [inline src/tools/dart-devtools.md]: It's not quite clear to me what this sentence adds. Should I read is as saying that DevTools works for both IDE and command-line based developer workflows?

Comment (kwalrath) [reply src/tools/dart-devtools.md]: I took it from the ddt page, and understood it as meaning "this isn't an IDE". But your wording is better.

**Comment by kwalrath** [comment]
ptal

**Pull request merged**